aspnetcore/fundamentals/middleware/extensibility/sample/Middleware/MiddlewareFactory.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using Microsoft.AspNetCore.Http;
+using MiddlewareExtensibilitySample.Data;
+
+namespace MiddlewareExtensibilitySample.Middleware
+{
+    #region snippet1
+    public class BasicMiddlewareFactory : IMiddlewareFactory
```

Comment:
Do we need this at all?

Reply:
> using ninject

Yes, I see what u mean ... from the container not via a created instance. I'll hit this when I get back from the 🏋️ gym 💪 and ping u back.